ios/docs/Controls/Navigation.md

```diff
@@ -0,0 +1,42 @@
+# Navigation
+
+## Overview
+
+FluentUI introduces the `NavigationBar` and `NavigationController` classes as subclasses of their UIKit counterparts, `UINavigationBar` and `UINavigationController` respectively. These classes will handle rendering all the relevant information with a Fluent look and feel. This means that for the most part, you can tell your app to use the Fluent `NavigationController` (which already knows to use the Fluent `NavigationBar` by default) and things should work for the most part.
+
+Similar to UIKit, most of the logic for what a `NavigationBar` should display comes from `UINavigationItem` instances. FluentUI [extends this class](../../FluentUI/Navigation/UINavigationItem%2BNavigation.swift) to include new features, such as appearance, interactivity, and subtitles.
+
+### Appearance Examples
+
+| `NavigationBar.Style` | Example |
+|-|-|
+| `.primary` | ![Navigation-Style-Primary.png](.attachments/Navigation-Style-Primary.png) |
+| `.system` | ![Navigation-Style-System.png](.attachments/Navigation-Style-System.png) |
+| `.custom` | ![Navigation-Style-Custom.png](.attachments/Navigation-Style-Custom.png) |
+
+| `NavigationBar.TitleStyle` | Example |
+|-|-|
+| `.system` | ![Navigation-TitleStyle-System1.png](.attachments/Navigation-TitleStyle-System1.png) ![Navigation-TitleStyle-System2.png](.attachments/Navigation-TitleStyle-System2.png) |
+| `.leading` | ![Navigation-TitleStyle-Leading1.png](.attachments/Navigation-TitleStyle-Leading1.png) ![Navigation-TitleStyle-Leading2.png](.attachments/Navigation-TitleStyle-Leading2.png) |
+| `.largeLeading` | ![Navigation-Style-Primary.png](.attachments/Navigation-Style-Primary.png) |
+
+### More Customization Options
+
+By specifying an appropriate instance of `NavigationBarTitleAccessory`, you can indicate to users that the title can be pressed.
+
+You can also specify an optional `titleImage` with the associated navigation item.
+
+| Specifications | Example |
+|-|-|
+| Title down arrow with `titleImage` | ![Navigation-Accessory-Image-TitleDownArrow.png](.attachments/Navigation-Accessory-Image-TitleDownArrow.png)
+| Subtitle disclosure | ![Navigation-Accessory-SubtitleDisclosure.png](.attachments/Navigation-Accessory-SubtitleDisclosure.png)
+
+## Implementation
+
+### Source Code
+
+[Navigation folder](../../FluentUI/Navigation/)
```

Comment:
Can we these to absolute paths or even better Github links just in case things end up moving around in the future

Reply:
Also we might want to hold off on adding documentation until the very end because these links are currently returning 404 errors